src/utils/OneUI/__tests__/OneUI.spec.js

```diff
@@ -0,0 +1,40 @@
+import cssVarsPonyfill from 'css-vars-ponyfill';
+import OneUI from '../OneUI';
+
+jest.mock('css-vars-ponyfill');
+
+describe('OneUI loader that starts the ponyfill and attach the theme to DOM', () => {
+    it('should attach <link> element with theme URL to DOM tree', () => {
+        const themeURL = 'http://theme-cdn.com/mytheme.css';
+        document.body.innerHTML = '<head></head>';
+
+        OneUI.applyTheme(themeURL);
+
+        expect(document.getElementsByTagName('link')[0].href).toBe(themeURL);
+    });
+
+    it('should load the ponyfill if browser is IE11', () => {
```

Comment:
maybe add also a test to check that ponyfill is not loaded for Chrome for instance